pkg/app/couchbase.go

```diff
@@ -39,6 +39,8 @@ const cbReadyTimeout = 5 * time.Minute
 // Regex to extract result from cb query response
 var countResp = regexp.MustCompile(`(?m){"\$1":([\d]+)},`)
 
+const numPings = 1
```

Comment:
this can be grouped with the const initialised in line 37.

Reply:
Done @viveksinghggits